locales/en/messages.json

```diff
@@ -1014,6 +1014,21 @@
     "initialSetupInstrumentsHead": {
         "message": "Instruments"
     },
+    "initialSetupInfoAPIversion": {
+        "message": "MSP API:"
+    },
+    "initialSetupInfoBuild": {
+        "message": "Firmware info"
+    },
+    "initialSetupInfoBuildinfo": {
+        "message": "Build date:"
+    },
+    "initialSetupInfoBuildOptions": {
+        "message": "Build Options:"
+    },
+    "initialSetupInfoBuildOptionsEmpty": {
+        "message": "Self Build - no Defines"
```

Comment:
Maybe Custom Build - or we just hide the section in that case.
What does Core Build look like?

Reply:
coreBuild are "Core Only"
I think for initialSetupInfoBuildOptionsEmpty "Local Build - no Defines" are right - changed to that